Change in-editor documentation style to match online docs.

bruvzg opened the pull request:
Some minor in-editor documentation style changes:

- Adds bullet to the property, constant, enum and method descriptions.
![consts_props](https://user-images.githubusercontent.com/7645683/82547849-0b303e00-9b63-11ea-97ee-a5d199109e6a.png)
- Rearranges some newlines.
![enums](https://user-images.githubusercontent.com/7645683/82551430-dcb56180-9b68-11ea-8bc8-a39a189c7eae.png)
- Fixes inheritance and enum line font size mismatch.
![inh](https://user-images.githubusercontent.com/7645683/82547867-11beb580-9b63-11ea-827e-0746659129a3.png)

bruvzg committed:
Change in-editor documentation style to be closer to online docs.

pycbouh commented:
Yay for consistency! But what's up with the `enum` keyword on the second one? Is it that small on purpose?

bruvzg commented:
> Is it that small on purpose?

No idea, I have not changed it, it was always small for me. If it's intended to be small, it's probably HiDPI scaling issue (and wrong inheritance size might be the same issue).

pycbouh commented:
Probably is a HiDPI issue, looks fine to me:

![image](https://user-images.githubusercontent.com/11782833/82550736-b04d1580-9b67-11ea-886e-d02bdf484422.png)
![image](https://user-images.githubusercontent.com/11782833/82550860-de325a00-9b67-11ea-9d9f-10c8d97314ed.png)

bruvzg commented:
> Probably is a HiDPI issue, looks fine to me

Yep, it's using default font instead of docs, updated PR.

Calinou commented:
For extra consistency, the `—` (en dash) character could be used to separate enum/constant values and their descriptions.

bruvzg commented:
*Update:* Added cell border/background color support to the RichTextLabel (disable by default), and some table borders to the documentation (overview and properties, as the separate commit).

![tables](https://user-images.githubusercontent.com/7645683/82565284-d502b680-9b82-11ea-9ac8-8f89a633fe56.png)
![props](https://user-images.githubusercontent.com/7645683/82565287-d59b4d00-9b82-11ea-8493-2e24399fc013.png)

> For extra consistency, the — (em dash) character could be used to separate enum/constant values and their descriptions. If it's too long, then – (en dash) can be used instead.

Changed it to en dash.

pycbouh commented:
> table borders to the documentation (overview and properties, as the separate commit)

I like the idea, but from your screenshots on my display it's barely noticeable.

bruvzg commented:
Added some padding (and cell padding support to the RichTextLabel), and changed colors to the editor theme default background variants.

![def_themes](https://user-images.githubusercontent.com/7645683/82572580-3e87c280-9b8d-11ea-8b1a-eb206420d2bf.png)

akien-mga commented on scene/gui/rich_text_label.h:
```diff
@@ -132,12 +132,23 @@ class RichTextLabel : public Control {
 			parent_frame = nullptr;
 			cell = false;
 			parent_line = 0;
+			odd_row_bg = Color(0, 0, 0, 0);
```
More initialization to the declaration.

akien-mga commented:
While the added RTL features look nice, I think they might warrant a dedicated PR as it's a core change, and it will conflict with #39248.
The rest of the doc style changes should be mergeable without second thought already.

bruvzg commented:
>  I think they might warrant a dedicated PR as it's a core change

Done, RTL related changes are in #39587

akien-mga commented:
Thanks!
